### DISCUSSION TITLE
Add support for query string in email URL field

### DISCUSSION THREAD

**Discussion by jean-gui** [craftcms/cms#10193] [Ideas] [answered]
Hi,

It would be useful if we could use values such as `foo@bar.com?Subject=Test` in a [URL field](https://github.com/craftcms/cms/discussions/new?category=ideas) that acts as an email.

Thanks!

**Comment by brandonkelly** [comment, marked as answer]
Just added this for the next release! (70398b35bbe765ac46bf08427298cd4443d09127)

**Reply by brandonkelly** [reply]
3.7.24 is out now with that change.

**Reply by oblongd** [reply]
Hi @brandonkelly,

I believe this update (or another recent update) might have introduced an unintended side effect. 

A client of mine just reported that adding or editing an external website link via a URL Field results in some new encoding being added when saved.

The issue arises when the URL contains a second URL inside a query parameter, and that URL also has query parameters. For example:

`https://proxy.lcls.org:2443/login?url=https://link.gale.com/apps/TGLF?u=edwa30873`
is converted to
`https://proxy.lcls.org:2443/login?url=https%3A//link.gale.com/apps/TGLF%3Fu%3Dedwa30873`

Unfortunately, this breaks the link through the proxy server, resulting in a 404.

Is there a config setting that can be used to disable URL encoding, or is this something that potentially needs to be addressed as a bugfix?

Thanks!

**Reply by brandonkelly** [reply]
@oblongd  Yeah we started encoding query string params alongside this change, but we only really needed to encode spaces to `+`, so I just reverted that change and went with a simple ` `→ `+` string replace instead (5b65fcdec1ee8dd2fc8b9c566455fee9bac9add0).

To get that change early, change your `craftcms/cms` requirement in `composer.json` to `"dev-develop as 3.7.25.1"` and run `composer update`.

**Reply by oblongd** [reply]
Thanks, Brandon... I truly appreciate the fast response and quick fix. 

Keep up the great work!

**Reply by brandonkelly** [reply]
Craft 3.7.26 is out with that change.

**Comment by oblongd** [comment]
Excellent! Thank you so much for the follow-up.
